docs/guides/user-containers/readme.md

```diff
@@ -341,13 +341,78 @@ reached before the container successfully exits, it triggers an `Error` status.
 timeout begins upon entry into the PostRun phase, allowing the containers the specified period to
 execute before the workflow enters an `Error` status.
 
+#### Long-Running Processes
+
+Some containers may run applications that are intended to run indefinitely (for example, an HTTP
+server listening for requests). The copy-offload user container is one such example. These
+long-running containers need a mechanism to be stopped; otherwise, the `postRunTimeoutSeconds`
+timeout will be reached, resulting in a workflow error.
+
+During the PostRun phase, if containers are still running, the NNF software will attempt to send an
+HTTPS POST request to the `/shutdown` endpoint on each user container. This process will continue
+until the container receives the request and gracefully exits, or until the timeout is reached.
+
+The software inside the container **must** be able to handle this request. The copy-offload user
+container includes this functionality. If this functionality is not present, the container will need
+to be terminated by some other means outside the container (for example, by the compute node
+application when it is time to stop). The request is defined in the next section.
+
+Alternatively, if `postRunTimeoutSeconds` is set to 0, the container exit codes will not be checked.
+The software will ignore the result of the containers and proceed to the Teardown phase, where the
+containers will be destroyed. This can be useful for long-running processes where the exit code is
+not important.
+
+#### Shutdown Request
+
+The request is sent using HTTPS (TLS required; client verifies server using a CA certificate from
+the Kubernetes `nnf-dm-usercontainer-server-tls` secret).
+
+The token is taken from the workflow specific token that is generated by the NNF software, if
```

Comment:
See the  `requires` => `user-container-auth` notes in https://nearnodeflash.github.io/latest/guides/user-interactions/readme/#command-arguments, and somehow talk about how the user can request a token for their container by adding `requires=user-container-auth` to the DW directive.

If they didn't ask for `requires=user-container-auth` then their container won't mount the Secret that contains the cert, and they won't get a per-workflow token.